docs/handbook/01_intro.md

```diff
@@ -0,0 +1,19 @@
+# Introduction
+
+Quantum computing has the potential to revolutionize many fields in the 21st century, such as cryptography {cite:p}`shorPolynomialtimeAlgorithmsPrime1997`, finance {cite:p}`eggerQuantumComputingFinance2020`, chemistry {cite:p}`mcardleQuantumComputationalChemistry2020`, machine learning {cite:p}`huangProvablyEfficientMachine2022`, and optimization {cite:p}`harwoodFormulatingSolvingRouting2021`. Over the past decade, numerous quantum computers from multiple providers based on different qubit technologies have been made publicly available.
+However, the best hardware is only as good as the software available to realize corresponding applications on it---a lesson learned from the past decades of research on designing and developing classical circuits and systems.
```

Comment:
Might be a little too harsh on experimentalists :P I'd propose something along the lines of:
However, as the quality of hardware increases, the software available to realize corresponding applications and automated solutions become increasingly important.

Reply:
(not totally happy with that though)

Reply:
In my opinion, it is a valid statement that software and hardware (need to) go hand in hand.
I guess without a strong proposal, I would rather like to keep the sentence as it is.

Reply:
Yes I agree, the sentence just sounded a little like the hardware is completely useless without software, I meant to 'soften' that a bit